examples/basic_usage/README.md

```diff
@@ -8,8 +8,30 @@ followed by the `terraform apply` command.
 Note that this example may create resources which cost money. Run
 `terraform destroy` when you no longer need these resources.
 
+## Requirements ##
+
+No requirements.
+
+## Providers ##
+
+No providers.
+
+## Modules ##
+
+| Name | Source | Version |
+|------|--------|---------|
+| cert\_role | ../../ |  |
+
+## Resources ##
+
+No resources.
+
+## Inputs ##
+
+No inputs.
+
 ## Outputs ##
 
 | Name | Description |
 |------|-------------|
-| role_arn | The ARN corresponding to the IAM role to be used for reading certificate data for the specified hostname |
+| role\_arn | The ARN corresponding to the IAM role to be used for reading certificate data for site.example.com |
```

Comment:
```suggestion
| role\_arn | The ARN corresponding to the IAM role to be used for reading certificate data for "site.example.com". |
```

Reply:
See commit 3550618.  Again, I don't think anything is added by the quotes so I omitted them.

Reply:
@jsf9k I don't see this change in the current version of the file. That commit also does not include this change. Did I miss something?

Reply:
Ah, I see it now.  I am blind today.

I think putting `site.example.com` there is misleading.  How about "The ARN corresponding to the IAM role to be used for reading certificate data for the host whose name is specified in the hostname input variable."

Reply:
That works for me.

Reply:
See commit 850648c.